scripts/run-nat-node.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+function echoerr() { echo "$@" 1>&2; }
+
+if [ $(id -u) -ne 0 ]; then 
+  echoerr "ERR: Please run as root!"
+  exit 1 
+fi
+
+
+declare rundir="/var/hoprd"
+unset RELEASE API_TOKEN
+
+function usage {
+  cat <<EOF
+Usage: $(basename "$0") [OPTION]
+
+  -r VALUE    relase name, e.g. prague
+  -t VALUE    API token
+  -h          display help
+EOF
+
+  exit 2
+}
+
+
+while getopts "r:t:h" optKey; do
+  case "$optKey" in
+    r)
+      RELEASE=$OPTARG
+      ;;
+    t)
+      API_TOKEN=$OPTARG
+      ;;
+    h|*)
+      usage
+      ;;
+  esac
+done
+
+shift $((OPTIND - 1))
+
+[ -z "$RELEASE" ] && usage
+[ -z "$API_TOKEN" ] && usage
+
+
+cat <<EOF >docker-compose.yaml
```

Comment:
let's move this into its separate file with a good name, easier to maintain and re-use that way

Reply:
Some values need to be expanded to `docker-compose.yaml` file, so I prefer generate it in the script. I'm adding a `-g` option for generate docker-compose.yaml only.